core/src/test/scripts/test_data/data_clin_coldefs_lowercase_attribute.txt

```diff
@@ -1,8 +1,8 @@
 #Patient Identifier	Overall Survival (Months)	Overall Survival Status	Disease Free (Months)	Disease Free Status	Eggs
 #Identifier to uniquely specify a patient.	Overall survival in months since initial diagnosis.	Overall patient survival status.	Disease free (months) since initial treatment.	Disease free status since initial treatment.	Baked beans
 #STRING	NUMBER	STRING	NUMBER	STRING	STRING
-#1	1	1	1	1	42
-PATIENT_ID	OS_MONTHS	OS_STATUS	DFS_MONTHS	DFS_STATUS	SAUSAGE
+#1	1	1	1	1	1
```

Comment:
why back to 1?

Reply:
I recycled this file (previous `data_clin_coldefs_unknown_attribute.txt`) which was never used. It belonged to a unit test that was removed I think (unknown attribute in clinical header), but the file was not deleted .

It contained 42, which is of course [the answer to everything](https://simple.wikipedia.org/wiki/42_answer), but has nothing to do with this unit test.